tests/backup/backup_test.go

```diff
@@ -899,6 +899,241 @@ var _ = Describe("{ShareBackupWithUsersAndGroups}", func() {
 	})
 })
 
+var _ = Describe("{ShareBackupAndEdit}", func() {
+	numberOfUsers := 2
+	users := make([]string, 0)
+	backupNames := make([]string, 0)
+	userContexts := make([]context.Context, 0)
+	var contexts []*scheduler.Context
+	var customBucketName string
+	var customBackupLocationName string
+	var backupLocationUID string
+	var cloudCredUID string
+	var cloudCred1UID string
+	var CloudCredUidList []string
+	var appContexts []*scheduler.Context
+	var bkpNamespaces []string
+	var clusterUid string
+	var clusterStatus api.ClusterInfo_StatusInfo_Status
+	var credName string
+	var cred1Name string
+	bkpNamespaces = make([]string, 0)
+	providers := getProviders()
+	JustBeforeEach(func() {
+		StartTorpedoTest("ShareBackupAndEdit",
+			"Share backup with restore and fullaccess mode and edit the shared backup", nil, 0)
+		log.InfoD("Deploy applications")
+		contexts = make([]*scheduler.Context, 0)
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			taskName := fmt.Sprintf("%s-%d", taskNamePrefix, i)
+			appContexts = ScheduleApplications(taskName)
+			contexts = append(contexts, appContexts...)
+			for _, ctx := range appContexts {
+				ctx.ReadinessTimeout = appReadinessTimeout
+				namespace := GetAppNamespace(ctx, taskName)
+				bkpNamespaces = append(bkpNamespaces, namespace)
+			}
+		}
+	})
+	It("Share large number of backups", func() {
+		Step("Validate applications and get their labels", func() {
+			log.InfoD("Validate applications and get their labels")
+			ValidateApplications(contexts)
+			log.Infof("Create list of pod selector for the apps deployed")
+		})
+		Step("Create Users", func() {
+			log.InfoD("Creating %d users", numberOfUsers)
+			var wg sync.WaitGroup
+			for i := 1; i <= numberOfUsers; i++ {
+				userName := fmt.Sprintf("testuser%v", i)
+				firstName := fmt.Sprintf("FirstName%v", i)
+				lastName := fmt.Sprintf("LastName%v", i)
+				email := fmt.Sprintf("testuser%v@cnbu.com", i)
+				wg.Add(1)
+				go func(userName, firstName, lastName, email string) {
+					err := backup.AddUser(userName, firstName, lastName, email, "Password1")
+					log.FailOnError(err, "Failed to create user - %s", userName)
+					users = append(users, userName)
+					wg.Done()
+				}(userName, firstName, lastName, email)
+			}
+			wg.Wait()
+		})
+		Step("Adding Credentials and Registering Backup Location", func() {
+			log.InfoD("Creating bucket, cloud credentials and backup location")
+			for _, provider := range providers {
+				customBucketName = fmt.Sprintf("%s-%v", getBucketName()[0], time.Now().Unix())
+				CreateBucket(provider, customBucketName)
```

Comment:
Why do you want a special bucket here? Can we use the common bucket?

Reply:
Changed the bucked to common bucket